### DISCUSSION TITLE
Sending one time photo.... option

### DISCUSSION THREAD

**Discussion by BroodPitt** [vasani-arpit/WBOT#255] [Ideas]
Hi!,

I was wondering if the WBOT supports one-time-photo option :) 

so i can send sensitive info and it only be viewed once :)

**Comment by vasani-arpit** [comment]
I think we can send text or image which are view once or as a disappearing message.

**Reply by BroodPitt** [reply]
And how do I activate that feature :-)

**Reply by vasani-arpit** [reply]
I need to develop that. well, I will add this to my list.
